pavelib/prereqs.py

```diff
@@ -132,10 +132,18 @@ def node_prereqs_installation():
     """
     Configures npm and installs Node prerequisites
     """
+    cb_error_text = "cb() never called"
     sh("test `npm config get registry` = \"{reg}\" || "
        "(echo setting registry; npm config set registry"
        " {reg})".format(reg=NPM_REGISTRY))
-    sh('npm install')
+
+    # Error handling around a race condition that produces "cb() never called" error. This
+    # ought to disappear when we upgrade npm to 3 or higher. TODO: clean this up when we do that.
+    capture = sh('npm install', capture=True, ignore_error=True)
```

Comment:
Re: ignore_error=True - I think that we don't want to swallow errors, e.g. trying to install a version that is not in the npm registry. Those should cause the packer task to fail.
Will that cause the npm install to return a non-zero return code? If so, I think that we should set ignore_error=False (the default). 
At any rate, it would be good to test. Either manually or automated. And as it looks like there are currently no tests in that area, we might just have to go with manual for expedience. :/